mod/isic/set/self/isic_section_formula.rb

```diff
@@ -0,0 +1,38 @@
+include_set Abstract::HardCodedFormula
+
+letter_start_finish = [%w[A 01 03],
+                       %w[B 05 09],
+                       %w[C 10 33],
+                       %w[D 35 35],
+                       %w[E 36 39],
+                       %w[F 41 43],
+                       %w[G 45 47],
+                       %w[H 49 53],
+                       %w[I 55 56],
+                       %w[J 58 63],
+                       %w[K 64 66],
+                       %w[L 68 68],
+                       %w[M 69 75],
+                       %w[N 77 82],
+                       %w[O 84 84],
+                       %w[P 85 85],
+                       %w[Q 86 88],
+                       %w[R 90 93],
+                       %w[S 94 96],
+                       %w[T 97 98],
+                       %w[U 99 99]]
```

Comment:
the gaps  between some of the letters are correct?

Reply:
Yes.  I found the reason:

> Due to the increased number of 2-digit categories, the divisions are coded
consecutively. However, some “gaps” have been provided to allow countries to introduce
division-level categories without a complete change of the ISIC coding. These
gaps have been introduced in sections where the need for additional divisions at the
national level is most likely to arise. For this purpose, the following division code
numbers have been left unused: 04, 34, 40, 44, 48, 54, 57, 67, 76, 83 and 89.